js/es-9-conditionAndString.mjs

```diff
@@ -0,0 +1,11 @@
+// Scrivete una funzione che riceve una stringa name e un numero num (può essere 0 o 1) e ritorna la strina "Hello" + name se num è 1 altrimenti "Bye" + name
+
+// sayHelloBye("alon", 1) ➞ "Hello Alon"
+
+const sayHello = (name, num) => {
+    num === 1 && console.log(`Hello ${name[0].toUpperCase()}${name.slice(1)}`);
+    num === 0 && console.log(`Bye ${name[0].toUpperCase()}${name.slice(1)}`);
+    // c'è sicuramente un altro modo per non ricopiare tutto il codice 
+    
```

Comment:
- usiamo sempre il return nei metodi e non più i console.log per far ritornare il valore
- non era il caso di inserire un if/else sul valore di num
- quando trovi ripetuta una logica, ti deve far venire in mente la possibilità di astrarlo in un metodo nuovo: prova a creare e usare una funzione che riceve una stringa e la ritorna con la prima lettera in maiuscolo